wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -3,24 +3,21 @@ import queryString from 'query-string';
 import io from "socket.io-client";
 import { Redirect } from "react-router-dom";
 
-import './Lobby.css';
-
-// change to http://localhost:5000 for local development
-const ENDPOINT  = 'http://localhost:5000';
+//Components
+import NoMatch from 'components/No Match/noMatch'; 
```

Comment:
Does this path `components/**No Match**/noMatch` cause any issues with the space? Might be a good idea to rename the folder without the space just to be safe.